src/tests/ftest/util/command_utils.py

```diff
@@ -685,6 +686,35 @@ def _get_result(self):
 
         return super(YamlCommand, self)._get_result()
 
+    def copy_certificates(self, source, hosts):
+        """Copy certificates files from the source to the destination hosts.
+
+        Args:
+            source (str): source of the certificate files.
+            hosts (list): list of the destination hosts.
+        """
+        yaml = self.yaml
+        while isinstance(yaml, YamlParameters):
+            if hasattr(yaml, "get_certificate_data"):
+                data = yaml.get_certificate_data(
+                    yaml.get_attribute_names(LogParameter))
+                for name in data:
+                    run_command(
+                        "clush -S -v -w {} /usr/bin/mkdir -p {}".format(
+                            ",".join(hosts), name))
+                    for file_name in data[name]:
+                        src_file = os.path.join(source, file_name)
+                        dst_file = os.path.join(name, file_name)
+                        run_command(
+                            "clush -S -v -w {} --copy {} --dest {}".format(
+                                ",".join(hosts), src_file, dst_file))
```

Comment:
To support testing with invalid certificate filenames, this code should log a warning instead of raising an exception:
```suggestion
                        result = run_command(
                            "clush -S -v -w {} --copy {} --dest {}".format(
                                ",".join(hosts), src_file, dst_file),
                            raise_exception=False)
                        if result.exit_status != 0:
                            self.log.info("WARNING: failure copying '%s' to '%s' on %s", src_file, dst_file, hosts)
```